plugins/lando-core/tasks/start.js

```diff
@@ -13,12 +13,18 @@ module.exports = lando => {
       // Start it if we can!
       if (app) {
         console.log(lando.cli.makeArt('appStart', {name: app.name, phase: 'pre'}));
+        // Normal bootup
         return app.start().then(() => {
           const status = utils.getStatusChecks(app);
           const type = !_.every(_.values(status)) ? 'report' : 'post';
           console.log(lando.cli.makeArt('appStart', {name: app.name, phase: type, checks: status}));
           console.log(lando.cli.formatData(utils.startTable(app), {format: 'table'}, {border: false}));
           console.log('');
+        })
+        // Provide help if there is an error
+        .catch(err => {
+          lando.log.debug(err);
+          console.log(lando.cli.makeArt('appStart', {phase: 'error'}));
```

Comment:
No catch for `plugins/lando-core/tasks/rebuild.js`?

Reply:
There might be at some point but the content for that should be thought out a bit more first

Reply:
Hopefully before May 4?

Reply:
We'll see, its not nearly as high a priority as providing helpful remediation steps for start/restart failures